test/pan_test2/rev-pan-test.py

```diff
@@ -33,104 +33,104 @@
-        "machine" : "[0:RV64GP]",                     # Core:Config; RV64GP for core 0
-        "startAddr" : "[0:0x0001014c]",               # Starting address for core 0
-        "memCost" : "[0:1:10]",                       # Memory loads required 1-10 cycles
-        "program" : os.getenv("REV_EXE", "pan_test.exe"),  # Target executable
-        "pan_nic" : "revcpu.PanNet",                  # Use the PAN NIC
-        "enable_pan" : 1,                             # Enable the internal RevNIC
-        "enable_test" : 0,                            # Disable the PAN test harness
-        "enable_pan_stats" : 1,                       # Enable the PAN statistics
-        "enableRDMAMbox" : 0,                         # Disable the RDMA Mailbox
-        "msgPerCycle" : 5,                            # Number of messages per cycle
-        "splash" : 1                                  # Display the splash message
+        "verbose": 5,                                     # Verbosity
+        "numCores": 1,                                    # Number of cores
+        "clock": "1.0GHz",                                # Clock
+        "memSize": 1024*1024*1024,                        # Memory size in bytes
+        "machine": "[0:RV64GP]",                          # Core:Config; RV64GP for core 0
+        "startAddr": "[0:0x0001014c]",                    # Starting address for core 0
+        "memCost": "[0:1:10]",                            # Memory loads required 1-10 cycles
+        "program": os.getenv("REV_EXE", "pan_test.exe"),  # Target executable
+        "pan_nic": "revcpu.PanNet",                       # Use the PAN NIC
+        "enable_pan": 1,                                  # Enable the internal RevNIC
+        "enable_test": 0,                                 # Disable the PAN test harness
+        "enable_pan_stats": 1,                            # Enable the PAN statistics
+        "enableRDMAMbox": 0,                              # Disable the RDMA Mailbox
+        "msgPerCycle": 5,                                 # Number of messages per cycle
+        "splash": 1                                       # Display the splash message
 })
 
 # setup the NICs
-nic0 = host_cpu0.setSubComponent("pan_nic", "revcpu.PanNet")  #-- host
-nic1 = pan_cpu1.setSubComponent("pan_nic", "revcpu.PanNet")  #-- pan
+nic0 = host_cpu0.setSubComponent("pan_nic", "revcpu.PanNet")  # -- host
+nic1 = pan_cpu1.setSubComponent("pan_nic", "revcpu.PanNet")   # -- pan
 
-iface0 = nic0.setSubComponent("iface", "merlin.linkcontrol")  #-- host
-iface1 = nic1.setSubComponent("iface", "merlin.linkcontrol")  #-- pan
+iface0 = nic0.setSubComponent("iface", "merlin.linkcontrol")  # -- host
+iface1 = nic1.setSubComponent("iface", "merlin.linkcontrol")  # -- pan
 
 # setup the router
 router = sst.Component("router", "merlin.hr_router")
 router.setSubComponent("topology", "merlin.singlerouter")
 
 # set the network params
-#-- host
+# -- host
```

Comment:
`flake8` expects at least one space between the `#` and the start of the comment's text.